packaging/suse/conf/portus_crono.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=A time-based background job scheduler daemon
+After=apache2.service
```

Comment:
mariadb can be in another server thus I would instead propose to have network running.